.changeset/short-ravens-clap.md

```diff
@@ -0,0 +1,5 @@
+---
+"pnpm": patch
```

Comment:
@zkochan This is probably major, but it's your decision.

Reply:
ok